tools/urls-check

```diff
@@ -123,9 +123,9 @@ def check_urls(verbose):
             if resp.geturl() != url and not any(fnmatch.fnmatch(url, pattern) for pattern in KNOWN_REDIRECTS):
                 redirects.append(url)
             if resp.getcode() >= 400:
-                failed.append(url)
-        except urllib.error.URLError:
-            failed.append(url)
+                failed.append(f"{url} : {resp.getcode()} {resp.reason}")
```

Comment:
As this looks a bit unusual/asymmetric, I tested this code path as well, by changing the 400 to 200, and then got:
```
Checked 177 URLs out of which 1 is/are invalid:
    https://www.dmtf.org/sites/default/files/standards/documents/DSP0134_2.7.1.pdf: 200 OK
```